docs/en/ingest-management/agent-policies.asciidoc

```diff
@@ -1,214 +1,215 @@
-[[agent-policies]
+[[agent-policy]
 [role="xpack"]
-= Policies
+= {agent} policies
+
+++++
+<titleabbrev>Policies</titleabbrev>
+++++
 
 beta[]
 
-[discrete]
-[[agent-policy]]
-== {policy}
-
-What is a {policy}?
-An {policy} is a collection of inputs and relating settings that comprise what data
-will be sent by the processes being run by {agent}. An {agent} can only be enrolled
-to a single {policy} at one time. In more details, an {policy} typically consists of
-a set of individual Integration Policies (the inputs), which is where the actual
-settings reside. The available settings in an Integration relate to the version of
-the Integration in use, newer functionality can be exposed in this manner over time.
-A {policy} can be represented in plain-text yaml format and is sent down to the
-{agent} as such. A {policy} can be viewed in kib in the {fleet}} 'Policies' section
-in the Policies list.  When viewing an individual {policy} details will show as to
-which Integrations are configured and more.
+A {policy} is a collection of inputs and settings that defines the data to be collected
+by {agent}. Each {agent} can only be enrolled in a single {policy}.
 
-[discrete]
-[[policy-use]]
-== How to Use {policy}
+Within an {agent} {policy} is a set of individual integration policies.
+These integration policies define the settings for each input type.
+The available settings in an integration relate to the version of
+the integration in use–newer functionality can be exposed in this manner over time.
```

Comment:
looking at 'use-newer' I feel like maybe a ; would work better?  It confused me when I read it.

Reply:
```suggestion
the integration in use; newer functionality can be exposed in this manner over time.
```